Automattic/jetpack
# CI: Sync changelogger version and package version constants

#### Changes proposed in this Pull Request:

Ensure that the version indicated by changelogger matches version constants in the code.

Then extend this to a composer `.extra.branch-alias.dev-monorepo` so we can use that in cross-project dependencies instead of `@dev` everywhere, with Jetpack's dev-dependency on changelogger as the test case.

#### Jetpack product discussion


p9dueE-2qL-p2
1199658893173824-as-1199910496214526

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* See that things work.
* Try changing versions in changelogger (either the version constant or the branch-alias), see that the scripts run by CI indicate failure.
* Try changing the branch-alias in changelogger and see that composer in Jetpack starts complaining about the mismatch.
